gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -149,53 +173,77 @@ export const wrapCommand = (command) => {
         proposalState,
       }
 
-      const isAlreadyExecuted = proposalState.didExecute
-      if (isAlreadyExecuted) throw new Error('Multisig Proposal is already executed')
-
-      this.require(
-        await this.isSameProposal(proposal, rawTx),
-        'The transaction generated is different from the Multisig Proposal provided',
-      )
+      if (proposalState.isExecuted)
```

Comment:
should this be `!proposalState.isExecuted`? Seems like before, we'd error if the proposal was already executed

Reply:
Oh nice catch, you're totally right!